### PR TITLE
Merge branch 'release-candidate-1.4' into bug-erms-2171-fuer1.4

### PR DESCRIPTION
**Pull request by anjaalbinnrw**
# Conflicts:
#	app/application.properties

### CODE REVIEW

File: app/application.properties [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-app.buildNumber=7944
+#Mon Jun 15 09:18:37 CEST 2020
+app.buildDate=15.06.2020 09\:18
+app.buildNumber=7949
 app.buildProfile=development
 app.grails.version=2.5.6
 app.name=laser
 app.servlet.version=3.0
-app.version=1.4.1
+app.version=1.4-RC
```
Comment (Moetez16) [inline app/application.properties:9]: app.version=1.4.2 ändern bitte

File: app/grails-app/controllers/com/k_int/kbplus/AjaxController.groovy [hunk 2]
```diff
@@ -2644,6 +2644,71 @@ class AjaxController {
...
+                    model.typeId = addressInstance.type?.id
+            }
+
+            model.modalText = message(code: 'default.edit.label', args: [message(code: messageCode)])
+            model.modalMsgSave = message(code: 'default.button.save_changes')
+            model.url = [controller: 'address', action: 'edit']
+            render template:"../templates/addresses/formModal", model: model
+        } else {
+            flash.error = "Diese Adresse existiert nicht (mehr)."
+            redirect(url: request.getHeader('referer'))
```
Comment (klodav) [inline app/grails-app/controllers/com/k_int/kbplus/AjaxController.groovy:2685]: Redirect within an ajax call ?

File: app/grails-app/views/templates/cpa/_person_full_details.gsp [hunk 3]
```diff
@@ -52,10 +52,15 @@
...
-                               value="${message(code: 'person.addresses.label')}">
-                        <g:render template="/address/formModal" model="['prsId': person.id, prId: personRole.id, modalId: 'addressFormModal' + personRole.id]"/>
+                        <% Map model = [:]
+                        model.prsId = person?.id
+                        model.redirect = '.'
+                        model.modalId = 'addressFormModal'+ personRole.id
+                        model.hideType = true%>
+                        <input class="ui mini icon button" type="button"
+                               value="${message(code: 'person.addresses.label')}"
+                               onclick="addresscreate('${model}');"
```
Comment (klodav) [inline app/grails-app/views/templates/cpa/_person_full_details.gsp:62]: function addresscreate() in organisation/show.gsp ?

File: app/grails-app/views/templates/addresses/_formModal.gsp [hunk 4]
```diff
@@ -0,0 +1,169 @@
+<%@ page import="de.laser.helper.RDStore; com.k_int.kbplus.Address;com.k_int.kbplus.RefdataValue;com.k_int.kbplus.RefdataCategory;de.laser.helper.RDConstants" %>
+<semui:modal id="addressFormModal" text="${modalText}" msgSave="${modalMsgSave}">
+    <g:form id="create_address_${modalId}" class="ui form" url="${url}" method="POST">
+        <g:if test="${addressInstance}">
+            <input type="hidden" name="id" value="${addressInstance.id}"/>
+        </g:if>
+        <input type="hidden" name="redirect" value="true"/>
+        <input id="type_${modalId}" name="type.id" type="hidden" value="${typeId}"/>
```
Comment (klodav) [inline app/grails-app/views/templates/addresses/_formModal.gsp:8]: Mismatch? 
typeId: RDStore.ADRESS_TYPE_POSTAL in organisation/identifier.gsp != \<input id="type_${modalId}" name="type.id" type="hidden" value="${typeId}"/>

**Comment by anjaalbinnrw** [comment]
Wegen größerem Refactoring gibts nun einen PR nach dev

**Pull request closed without merging**